{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-13006",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/about.gitlab.com\/blog\/categories\/releases\/",
                    "name": "https:\/\/about.gitlab.com\/blog\/categories\/releases\/",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https:\/\/about.gitlab.com\/releases\/2019\/07\/03\/security-release-gitlab-12-dot-0-dot-3-released\/",
                    "name": "https:\/\/about.gitlab.com\/releases\/2019\/07\/03\/security-release-gitlab-12-dot-0-dot-3-released\/",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "An issue was discovered in GitLab Community and Enterprise Edition 9.0 and through 12.0.2. Users with access to issues, but not the repository were able to view the number of related merge requests on an issue. It has Incorrect Access Control."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2020-03-10T17:15Z",
    "lastModifiedDate": "2020-03-10T17:30Z"
}